libs/host/GarnetServer.cs

```diff
@@ -76,7 +77,7 @@ public class GarnetServer : IDisposable
         /// </summary>
         /// <param name="commandLineArgs">Command line arguments</param>
         /// <param name="loggerFactory">Logger factory</param>
-        public GarnetServer(string[] commandLineArgs, ILoggerFactory loggerFactory = null, bool cleanupDir = false)
+        public GarnetServer(string[] commandLineArgs, IAuthenticationSettings authenticationSettingsOverride = null, ILoggerFactory loggerFactory = null, bool cleanupDir = false)
```

Comment:
should we not create a new constructor for this or add it at the end atleast?
This will break for anyone already using existing optional parameters ( ex: an ILoggerFactory instance as 2nd parameter)

Reply:
Thanks. Need to keep in mind for compatibility.